pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools"]
```

Comment:
Since the package use setuptools-scm for versioning you need to add it there too.

Reply:
Setuptools has a set of options that we have added them in *python.cfg* such as
```
setup_requires = setuptools>=64; wheel; setuptools_scm[toml]>=3.4.3
```
Having that, do we have to still add them in *pyproject.toml*?

Reply:
Yes since we are removing setup.cfg entirely, basically all those should move to the require section of the pyproject.toml

Reply:
> Yes since we are removing setup.cfg entirely, basically all those should move to the require section of the pyproject.toml

👍 💯